### PR TITLE
A11y updates: use IronMenubarBehavior: select on focus, prevent scrolling on arrow keys.

### PR DESCRIPTION
**Pull request by bicknellr**

### CODE REVIEW

**Comment by bicknellr** [comment]
cc @robdodson

**Comment by robdodson** [comment]
Cc @ebidel

File: paper-radio-group.html [hunk 1]
```diff
@@ -157,34 +156,32 @@
...
     },
 
-    /**
-     * Selects the next item. If the next item is disabled, then it is
-     * skipped, and the next item after it is selected.
-     */
-    selectNext: function() {
-      var length = this.items.length;
-      var newIndex = Number(this._valueToIndex(this.selected));
+    _onUpKey: function(event) {
```
Comment (valdrinkoshi) [inline paper-radio-group.html]: you basically want `keyup` to behave like `keyleft`, so why not just doing:
```javascript
_onUpKey: function(e) { this._onLeftKey(e); },
_onDownKey: function(e) { this._onRightKey(e); },
```

File: paper-radio-group.html [hunk 2]
```diff
@@ -157,34 +156,32 @@
...
+      this._focusNext();
+      this._activateFocusedItem();
+      event.detail.keyboardEvent.preventDefault();
+    },
 
-      this._itemActivate(this._indexToValue(newIndex), this.items[newIndex]);
+    _onLeftKey: function(event) {
+      Polymer.IronMenubarBehaviorImpl._onLeftKey.apply(this, arguments);
+      this._activateFocusedItem();
+      event.detail.keyboardEvent.preventDefault();
```
Comment (valdrinkoshi) [inline paper-radio-group.html]: not needed, as it's already done by the implementation of `_onLeftKey`  in `IronMenubarBehaviorImpl`

File: paper-radio-group.html [hunk 3]
```diff
@@ -157,34 +156,32 @@
...
     },
+
+    _onRightKey: function(event) {
+      Polymer.IronMenubarBehaviorImpl._onRightKey.apply(this, arguments);
+      this._activateFocusedItem();
+      event.detail.keyboardEvent.preventDefault();
```
Comment (valdrinkoshi) [inline paper-radio-group.html]: not needed, as it's already done by the implementation of `_onLeftKey`  in `IronMenubarBehaviorImpl`

File: paper-radio-group.html (hunk 1 above)
Comment (bicknellr) [inline paper-radio-group.html]: IronMenubarBehavior's handles switching between `_focusNext` and `_focusPrevious` for RTL

Comment (valdrinkoshi) [inline paper-radio-group.html]: to be more precise, you should also check for the direction (`this._isRTL`). What't the requirement for this one?
In the left-to-right scenario: left = up, and right = down. What about right-to-left?

Comment (bicknellr) [inline paper-radio-group.html]: Same situation as IronMenubarBehavior: right and left are swapped.

Comment (valdrinkoshi) [inline paper-radio-group.html]: While top and down should remain the same (be `isRTL`-independent), right?
In that case LGTM 👍

File: paper-radio-group.html [hunk 4]
```diff
@@ -157,34 +156,32 @@
...
-     * Selects the next item. If the next item is disabled, then it is
-     * skipped, and the next item after it is selected.
-     */
-    selectNext: function() {
-      var length = this.items.length;
-      var newIndex = Number(this._valueToIndex(this.selected));
+    _onUpKey: function(event) {
+      this._focusPrevious();
+      this._activateFocusedItem();
+      event.detail.keyboardEvent.preventDefault();
```
Comment (valdrinkoshi) [inline paper-radio-group.html]: nit: you can call `event.preventDefault()`, `iron-a11y-keys-behavior` will call it on the `keyBoardEvent` https://github.com/PolymerElements/iron-a11y-keys-behavior/blob/master/iron-a11y-keys-behavior.html#L456

**Comment by valdrinkoshi** [comment]
minor comments, LGTM 👍

File: paper-radio-group.html (hunk 1 above)
Comment (bicknellr) [inline paper-radio-group.html]: Yup! (until css `writing-mode` gets better support lul)

File: paper-radio-group.html (hunk 2 above)
Comment (bicknellr) [inline paper-radio-group.html]: removed

File: paper-radio-group.html (hunk 3 above)
Comment (bicknellr) [inline paper-radio-group.html]: removed

File: paper-radio-group.html (hunk 4 above)
Comment (bicknellr) [inline paper-radio-group.html]: nifty!

Comment (bicknellr) [inline paper-radio-group.html]: changed

Comment (valdrinkoshi) [inline paper-radio-group.html]: 💯

**Comment by valdrinkoshi** [comment]
:shipit:

File: bower.json [hunk 5]
```diff
@@ -20,9 +20,8 @@
   "homepage": "https://github.com/PolymerElements/paper-radio-group",
   "ignore": [],
   "dependencies": {
-    "iron-a11y-keys-behavior": "PolymerElements/iron-a11y-keys-behavior#^1.0.0",
-    "iron-selector": "PolymerElements/iron-selector#^1.0.0",
-    "polymer": "Polymer/polymer#^1.1.0"
+    "polymer": "Polymer/polymer#^1.1.0",
+    "iron-menu-behavior": "PolymerElements/iron-menu-behavior#^1.1.7"
```
Comment (valdrinkoshi) [inline bower.json]: does this need to be `v1.1.7`?

Comment (bicknellr) [inline bower.json]: Hrm, this is just what bower put there when I ran `bower install --save PolymerElements/iron-menu-behavior`... I don't think it should matter? The bigger issue might be that this relies on changes to IronSelector that aren't necessarily transitively 'included' by IronMenuBehavior's version. :/

**Pull request merged**